tests/unit/SidebarFilterTest.js

```diff
@@ -691,58 +705,6 @@ describe('Sidebar', () => {
     });
 
     describe('Archived chat', () => {
-        describe('in default (most recent) mode', () => {
-            it('is shown regardless if it has comments or not', () => {
```

Comment:
This is a behavior we are changing so I am removing this test. Empty chats should not be showing at all.